torch/csrc/jit/tensorexpr/ir_mutator.h

```diff
@@ -90,6 +91,9 @@ AT_FORALL_SCALAR_TYPES_AND2(Bool, Half, IMM_MUTATE_DECLARE);
   virtual Stmt* mutate(const Allocate* v);
   virtual Stmt* mutate(const Free* v);
   virtual Stmt* mutate(const Cond* v);
+
+  protected:
+  const Expr* DefaultMutator(const BaseCallNode* v, std::vector<const Expr*>& params);
```

Comment:
We're not consistent about naming currently but I think we should try to get back the fbcode convention of `lowerCamelCase` for methods.

Reply:
It's currently using this convention to match `BaseCallNode::DefaultMutator`.  I'm fine with changing the convention but I'd rather do that separately.